ext/reflection/php_reflection.c

```diff
@@ -1657,7 +1657,16 @@ ZEND_METHOD(ReflectionFunctionAbstract, getClosureScopeClass)
 	GET_REFLECTION_OBJECT();
 	if (!Z_ISUNDEF(intern->obj)) {
 		closure_func = zend_get_closure_method_def(Z_OBJ(intern->obj));
-		if (closure_func && closure_func->common.scope) {
+		
+		if (!closure_func) {
+		    return;
+		}
+		
+		if (closure_func->common.fn_flags & ZEND_ACC_ABSTRACT) {
+		    return;
+		}
+		
+		if (closure_func->common.scope) {
```

Comment:
I think both getClosureThis and getClosureScopeClass can be reasonably supported on partials, the same way they work on fake closures.

Reply:
I've added tests for this, but not sure we are returning the appropriate this_ptr, anyway, it looks so different that I'm gonna mark this resolved, have a look again in next review ...